src/components/atoms/Link/Link.tsx

```diff
@@ -1,36 +1,37 @@
 import React from 'react';
 import styled, { css } from 'styled-components';
-import { colors, TColors } from '../../../constants/colors';
+import { colors } from '../../../constants/colors';
 import { typography } from '../../../constants/typography';
 
 export interface ILinkProps
   extends React.AnchorHTMLAttributes<HTMLAnchorElement>,
     React.RefAttributes<HTMLAnchorElement> {
   target?: '_blank';
-  color?: TColors['actionPlain'] | TColors['white'];
+  negative?: boolean;
```

Comment:
The reason I chose a `boolean` prop rather than a `color` or `variant` one is that the **design team** wants to move away from text on top of dark background.

This is kept for legacy puprposes but probably de deprecated and removed in *v5* 💅🏻 🧽 🙇🏻‍♂️